ui/src/main/webapp/src/app/project/migration-project-form.component.html

```diff
@@ -1,6 +1,31 @@
 <div class="container-fluid">
     <form #projectForm="ngForm" (ngSubmit)="save()" class="form-horizontal wizard-form">
 
+        <style>
+            input:invalid {
+                border-color: #c00;
+                box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
+            }
+            input:invalid:focus {
+                border-color: #900;
+                box-shadow: inset 0 1px 1px rgba(0,0,0,.075), 0 0 6px #f33;
+            }
+            input:invalid.ng-untouched {
+                border-color: initial;
+                box-shadow: initial;
+            }
+
+            .error-msg-invalid { display: none; }
+            input:invalid ~ .error-msg-invalid {
+                color: #c00;
+                display: initial;
+            }
+            input:invalid.ng-untouched ~ .error-msg-invalid {
+                color: #c00;
+                display: none;
+            }
+        </style>
```

Comment:
Guys I really do not see the black box. It is red or blue. The red is not that dark to be confused with black. What browser do you use?